lab01-activity.Rmd

```diff
@@ -264,6 +280,11 @@ conda install -c bioconda samtools
 
 The human reference genome in fasta format can be found in the lab data directory, specifically ` /local-scratch/course_files/MBB243/hg38.fa`. Using what you learned in lecture, construct a command that uses samtools to extract the sequence from this region: `chr14:65102304-65102339` and store the sequence as a variable. Note that we don't want to store the "header" (starts with `>`). Construct this code in the code chunk below. Your code chunk should end with an `echo` command that prints the contents of your new variable.
 
+```{python}
+samtools faidx data/hg38.fa chr14:65102304-65102339
+echo
```

Comment:
Nice try, you just need to define it to a variable like:
chrom14=$(samtools faidx data/hg38.fa chr14:65102304-65102339)
echo $chrom14